.github/workflows/deploy.yml

```diff
@@ -0,0 +1,174 @@
+
+      - name: Build and push backend
+        id: docker_build_backend
+        run: |
+          docker build -f backend/sozisel/Dockerfile -t sozisel-backend backend/sozisel
+          docker tag sozisel-backend bombardierzy/sozisel-backend:latest
+          docker push bombardierzy/sozisel-backend:latest
+  
+  build_and_push_frontend:
+    needs:
+      - test-backend
+      - test-frontend
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      - name: Download schema file artifact
+        uses: actions/download-artifact@v2
+        with:
+          name: schema
+          path: frontend/sozisel-app
+
+      - name: Login to DockerHub
+        uses: docker/login-action@v1 
+        with:
+          username: ${{ secrets.DOCKER_USERNAME }}
+          password: ${{ secrets.DOCKER_PASSWORD }}
+
+      - name: Build and push frontend
+        id: docker_build_frontend
+        run: |
+          docker build -f frontend/sozisel-app/Dockerfile -t sozisel-frontend frontend/sozisel-app
+          docker tag sozisel-frontend bombardierzy/sozisel-frontend:latest
+          docker push bombardierzy/sozisel-frontend:latest
+
+  build_and_push_router:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      - name: Login to DockerHub
+        uses: docker/login-action@v1 
+        with:
+          username: ${{ secrets.DOCKER_USERNAME }}
+          password: ${{ secrets.DOCKER_PASSWORD }}
+
+      - name: Build and push router
+        id: docker_build_router
+        run: |
+          docker build -f router/Dockerfile -t sozisel-router router
+          docker tag sozisel-router bombardierzy/sozisel-router:latest
+          docker push bombardierzy/sozisel-router:latest
+
+  deploy_to_sozisel:
+    runs-on: ubuntu-latest
+    needs:
+      - build_and_push_backend
+      - build_and_push_frontend
+      - build_and_push_router
+    steps:
+      - uses: actions/checkout@v2
+      - name: Copy configuration
+        uses: appleboy/scp-action@master
+        with:
+          host: ${{ secrets.DEPLOY_HOST }}
+          username: ${{ secrets.DEPLOY_USERNAME }}
+          password: ${{ secrets.DEPLOY_PASSWORD }}
+          source: "deploy/*"
+          target: "services/sozisel"
+
+      - name: Executing remote ssh commands
+        uses: appleboy/ssh-action@master
+        with:
+          host: ${{ secrets.DEPLOY_HOST }}
+          username: ${{ secrets.DEPLOY_USERNAME }}
+          password: ${{ secrets.DEPLOY_PASSWORD }}
+          script: |
+            mv ~/services/sozisel/deploy/* ~/services/sozisel/
+            mv ~/services/sozisel/prod-docker-compose.yml ~/services/sozisel/docker-compose.yml
+            rm -r ~/services/sozisel/deploy
+            DOCKER_COMPOSE_APP_FILE=~/services/sozisel/docker-compose.yml
+            DOCKER_COMPOSE_JITSI_FILE=~/services/sozisel/jitsi-compose.yml
+            JITSI_ENV_FILE=~/services/sozisel/.jitsi_env
```

Comment:
I hope that the jitsi config has all its secrets regenerated for deploy purposes and does not reuse local tokens ect.?